damien/models/department.py

```diff
@@ -312,6 +313,26 @@ def to_api_json(
             feed['totalSections'] = self.row_count
         return feed
 
+    def uses_midterm_forms(self, term_id):
+        params = {
+            'department_id': self.id,
+            'term_id': term_id,
+        }
+        query = text(
+            """SELECT *
+                FROM department_forms df
+                JOIN department_catalog_listings dcl ON df.name = dcl.subject_area || '_MID'
+                WHERE df.name LIKE '%_MID'
+                AND df.deleted_at IS NULL
+                AND dcl.department_id = :department_id
+                AND (dcl.start_term_id IS NULL OR dcl.start_term_id <= :term_id)
+                AND (dcl.end_term_id IS NULL OR dcl.end_term_id > :term_id)
```

Comment:
@lyttam - to confirm:  we don't want `dcl.end_term_id >= :term_id` in the line above?

Reply:
@johncrossman I think you're right; will amend.